conda_build/build.py

```diff
@@ -428,42 +477,19 @@ def create_info_files(m, files, config, prefix):
     :type files: list of str
     '''
 
+    if on_win:
+        # make sure we use '/' path separators in metadata
+        files = [_f.replace('\\', '/') for _f in files]
```

Comment:
I think this coverage info just isn't being reported because of Appveyor build failures.  Looking past this gap in coverage.